tests/behavior/transforms/remap.toml

```diff
@@ -419,7 +419,7 @@
     .a = sha1(string!(.a))
 
     if sha1(string!(.b)) == "62cdb7020ff920e5aa642c3d4066950dd1f01f4d" {
-      .b = sha1(.a + .b + "baz")
+      .b = sha1(.a + string!(.b) + "baz")
```

Comment:
I'm a little curious how this was passing before.

Reply:
It looks like d1e8fbdd416e2ff9d0af27de6d4898ce11e4a58f fixed the type checking on this one.

Reply:
No worries, I think CI should be back in a good state now so it'll be easier to catch them. Workflow syntax errors are still very hard to notice though.